### PR TITLE
docs: Reword AMI tasks to options

### PR DESCRIPTION
**Pull request by chrisnegus**
**Description**: Updated the [Managing AMIs](https://karpenter.sh/preview/tasks/amitasks/) docs to change the name identifying the "tasks" to "options" instead. Also backported the update to all versions from v0.32 and higher.

**How was this change tested?**

**Does this change impact docs?**
- [X] Yes, PR includes docs updates 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/tasks/amitasks.md [hunk 1]
```diff
@@ -28,29 +28,29 @@ These include: Expiration (if node expiry is set, the node is marked for deletio
...
+* Option 2 (Lock down AMIs): If workloads require a particluar AMI, this option can make sure that it is the only AMI used by Karpenter. This can be used in combination with Option 1, where you lock down the AMI in production, but allow the newest AMIs in a test cluster while you test your workloads before upgrading production. Keep in mind that this makes upgrades a manual process for you.
+* Option 3 ([Disruption budgets]({{< relref "../concepts/disruption/" >}})): This option can be used as a way of mitigating the scope of impact if a new AMI causes problems with your workloads. With Disruption budgets you can slow the pace of upgrades to nodes with new AMIs or make sure that upgrades only happen during selected dates and times (using `schedule`). This doesn't prevent a bad AMI from being deployed, but it allows you to control when nodes are upgraded, and gives you more time respond to rollout issues.
 
-## Tasks
+## Task
```
Comment (jonathan-innis) [inline website/content/en/preview/tasks/amitasks.md:49]:
```suggestion
## Options
```

Consider calling this section options

File: website/content/en/preview/tasks/amitasks.md [hunk 2]
```diff
@@ -28,29 +28,29 @@ These include: Expiration (if node expiry is set, the node is marked for deletio
...
 
-The following tasks let you have an impact on Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
+The following task lays out the options you have to impact Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
```
Comment (jonathan-innis) [inline website/content/en/preview/tasks/amitasks.md:51]:
```suggestion
The following lays out the options you have to impact Karpenter’s behavior as it relates to how nodes are created and AMIs are consumed.
```

nit: personal preference on wording to make it less verbose

File: website/content/en/preview/tasks/amitasks.md [hunk 3]
```diff
@@ -77,9 +77,9 @@ For example, you could have:
...
     amiFamily: AL2
   ```
-* **Production clusters**: After you've confirmed that the AMI works in your lower environments, you can pin the latest AMIs to be deployed in your production clusters to roll out the AMI. One way to do that is to use `amiSelectorTerms` to set the tested AMI to be used in your production cluster. Refer to Task 2 for how to choose a particular AMI by `name` or `id`. Remember that it is still best practice to gradually roll new AMIs into your cluster, even if they have been tested. So consider implementing that for your production clusters as described in Task 3.
+* **Production clusters**: After you've confirmed that the AMI works in your lower environments, you can pin the latest AMIs to be deployed in your production clusters to roll out the AMI. One way to do that is to use `amiSelectorTerms` to set the tested AMI to be used in your production cluster. Refer to Option 2 for how to choose a particular AMI by `name` or `id`. Remember that it is still best practice to gradually roll new AMIs into your cluster, even if they have been tested. So consider implementing that for your production clusters as described in Option 3.
```
Comment (jonathan-innis) [inline website/content/en/preview/tasks/amitasks.md:80]: Should we provide links to the anchors in this doc when we are referencing other options in the doc?

File: website/content/en/preview/tasks/amitasks.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/preview/tasks/amitasks.md:49]: Done.

File: website/content/en/preview/tasks/amitasks.md (hunk 2 above)
Comment (chrisnegus) [reply website/content/en/preview/tasks/amitasks.md:51]: Sure. Done.

File: website/content/en/preview/tasks/amitasks.md (hunk 3 above)
Comment (chrisnegus) [reply website/content/en/preview/tasks/amitasks.md:84]: I added links to each of the options where it is introduced above. I think that should do it.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**